composer.json

```diff
@@ -4,7 +4,9 @@
     "type": "project",
     "license": "MIT",
     "require": {
-        "equip/framework": "^1.0"
+        "equip/framework": "^1.0",
+        "josegonzalez/dotenv": "^2.0",
```

Comment:
Figured since the .env file was already committed to the repo, it made sense to include the dotenv library and configuration.